README.md

```diff
@@ -1,2 +1,37 @@
-Automatic combat runner for ttrpg D&D. Project takes in JSON sourced combatants, converts them into functioning objects and pits them 
-against each other using D&D mechanics.
+# D&D Brawler Engine
+Automatic combat runner for ttrpg D&D. Project takes in JSON sourced combatants, converts them into functioning objects and pits them against each other using D&D mechanics. 
+
+## How To Use
+The project can currently be run in an IDE, the desired matchup is fed in as command line arguments. The results and the round-by-round combat will be outputted in the command line.
+
+The structure of the arguments is as follows:
+> \<monster name> <team [red/blue]> \<number of monsters>
+
+And example input might look like the following:
+> direwolf red 3 blackbear red 5 goblin blue 10 crocadile blue 2 skeleton blue 6 
+
+![image](https://user-images.githubusercontent.com/22297137/205460833-2d3b19e4-d05c-4040-a2c7-ea24fe037020.png)
+
+**Currently implemented monsters:**
+* Ankheg
+* Balor
+* Behir
+* Black Bear
+* Brown Bear
+* Crocodile
+* Direwolf
+* Flying Sword
+* Giant Eagle
+* Goblin
+* Green Dragon
+* Skeleton 
+* Troll
+
+## Output
+The results of the program wil be outputted to the console. Debugging information will be displayed in red text. The winner of the brawl and how many of the winning team will be shown, then the turn by turn battle information.
```

Comment:
Typo in "wil". Also the if you look at BrawlerOutputter class the output can be returned as either a JSON object with a specific structure in getBrawlOutput(), or as a String in getBrawlOutputAsString(). Can you update to reflect this pls?

Reply:
Good spot, thanks for that. Additional output information has been included.